talk/owt/BUILD.gn

```diff
@@ -317,14 +317,14 @@ static_library("owt_sdk_base") {
         "sdk/base/linux/msdkvideosession.h",
         "sdk/base/linux/vaapi_allocator.cc",
         "sdk/base/linux/vaapi_allocator.h",
+        "sdk/base/linux/xwindownativeframe.h",
         "sdk/base/windowcapturer.cc",
       ]
       defines += [ "OWT_USE_MSDK" ]
     }
     sources += [
       "sdk/base/linux/videorenderlinux.cc",
```

Comment:
videorendererlinux.cc includes xwindownativeframe.h

Any reason .h is controlled by OWT_USE_MSDK while the renderer does not?

Reply:
It's the same behavior as the old cg branch. videorendererlinux.cc includes xwindownativeframe.h only when OWT_USE_MSDK is defined. If it's not defined, it uses an empty onFrame implementation.